pkg/runctl/controller.go

```diff
@@ -769,13 +812,30 @@ func (c *Controller) addToWorkqueueFromAssociated(obj interface{}) {
 			utilruntime.HandleError(fmt.Errorf("error decoding object tombstone, invalid type"))
 			return
 		}
-		klog.V(3).Infof("Recovered deleted object '%s' from tombstone", object.GetName())
+		klog.V(3).InfoS("Recovered deleted object from unknown state",
+			"recoveredObject", object.GetNamespace()+"/"+object.GetName())
 	}
-	klog.V(4).Infof("Processing object: %s", object.GetSelfLink())
+	klog.V(4).InfoS("Processing the object", "object", object.GetNamespace()+"/"+object.GetName())
```

Comment:
See above

Reply:
Pls refer [the comment](https://github.com/SAP/stewardci-core/pull/387#discussion_r1238272323) and verify the change adfbc2abfdf982d656d0b4de583541d07ca5e8a2.